js/modules/loki_snode_api.js

```diff
@@ -84,6 +78,32 @@ class LokiSnodeAPI {
     }
   }
 
+  async getSwarmNodesForPubKey(pubKey) {
+    let conversation;
+    let swarmNodes;
+    try {
+      conversation = ConversationController.get(pubKey);
```

Comment:
You don't need `conversation` outside the try bock:
```suggestion
      const conversation = ConversationController.get(pubKey);
```